src/bp-moderation/bp-moderation-admin.php

```diff
@@ -61,5 +70,148 @@ function bp_moderation_admin_menu_order( $custom_menus = array() ) {
  * @since BuddyBoss 1.5.4
  */
 function bp_moderation_admin_load() {
+	global $bp_moderation_list_table;
+
+	$doaction = bp_admin_list_table_current_bulk_action();
+
+	/**
+	 * Fires at top of Moderation admin page.
+	 *
+	 * @since BuddyBoss 1.5.4
+	 *
+	 * @param string $doaction Current $_GET action being performed in admin screen.
+	 */
+	do_action( 'bp_moderation_admin_load', $doaction );
+
+	// Create the Moderation screen list table.
+	$bp_moderation_list_table = new BP_Moderation_List_Table();
+
+	// The per_page screen option.
+	add_screen_option( 'per_page', array( 'label' => __( 'Moderation Request', 'buddyboss' ) ) );
+
+	// Help panel - overview text.
+	get_current_screen()->add_help_tab(
+		array(
+			'id'      => 'bp-moderation-overview',
+			'title'   => __( 'Overview', 'buddyboss' ),
```

Comment:
@janakkaneriya007 Still I can found `__() `. Updated this with escaping function